airflow/providers/amazon/aws/executors/ecs/Setup_guide.md

```diff
@@ -0,0 +1,148 @@
+<!--
```

Comment:
A guide for setting up a set of resources to run the ECS executor. This is just one suggestion, there are many ways to do this but we thought it'd be helpful for users to have something to at least get started with.